src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -5636,7 +5633,8 @@ def test_aks_create_with_monitoring_legacy_auth(
 
             assert False
         except Exception:
-            pass  # this is expected
+            # do nothing as this is expected
```

Comment:
@evelyn-ys, @wangzelin007, seems `azdev scan/mask` still considers this as a credential, any idea how to bypass this false alert?

![image](https://github.com/user-attachments/assets/923dec47-536e-43fc-907f-8224f6e64c1f)